rockchip: Add Realtek RTL8821CU support for the Radxa ROCK Pi E v3

RadxaNaoki opened the pull request:
There is a variant of the Radxa ROCK Pi E v3 equipped with the Realtek RTL8821CU. Add the kmod-rtw88-8821cu package for it.

RadxaNaoki commented:
@1715173329 @hauke @maurerr Could you please review this PR?

stintel commented on target/linux/rockchip/image/armv8.mk:
```diff
@@ -222,7 +222,7 @@ define Device/radxa_rock-pi-e-v3
   SOC := rk3328
   DEVICE_DTS := rockchip/rk3328-rock-pi-e
-  DEVICE_PACKAGES := kmod-rtw88-8723du kmod-usb-net-cdc-ncm kmod-usb-net-rndis wpad-basic-mbedtls
+  DEVICE_PACKAGES := kmod-rtw88-8821cu kmod-rtw88-8723du kmod-usb-net-cdc-ncm kmod-usb-net-rndis wpad-basic-mbedtls
```
Please maintain alphabetical order.

RadxaNaoki replied:
Thanks. Fixed.

RadxaNaoki committed:
rockchip: Add Realtek RTL8821CU support for the Radxa ROCK Pi E v3

There is a variant of the Radxa ROCK Pi E v3 equipped with the Realtek
RTL8821CU. Add the kmod-rtw88-8821cu package for it.

Signed-off-by: FUKAUMI Naoki <naoki@radxa.com>
